src/Microsoft.DotNet.Cli.Utils/CommandResolution/ProjectDependenciesCommandResolver.cs

```diff
@@ -25,12 +25,12 @@ public class ProjectDependenciesCommandResolver : ICommandResolver
         {
             if (environment == null)
             {
-                throw new ArgumentNullException("environment");
+                throw new ArgumentNullException(nameof(environment));
```

Comment:
VS has started complaining about these as errors now :). I kinda like that, but it means I feel obligated to update them :P